src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/SupplierAssessmentDTO.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.SupplierAssessment
+import java.time.OffsetDateTime
+import java.util.UUID
+
+class SupplierAssessmentDTO(
+  val id: UUID,
+  val appointmentTime: OffsetDateTime?,
+  val durationInMinutes: Int?,
+  val createdAt: OffsetDateTime,
+  val createdBy: AuthUserDTO,
+) {
+  companion object {
+    fun from(supplierAssessment: SupplierAssessment): SupplierAssessmentDTO {
+      return SupplierAssessmentDTO(
+        id = supplierAssessment.id,
+        appointmentTime = supplierAssessment.appointment.appointmentTime,
+        durationInMinutes = supplierAssessment.appointment.durationInMinutes,
+        createdAt = supplierAssessment.appointment.createdAt,
```

Comment:
see my comment about createdAt/By elsewhere. let's not follow the same pattern here just to match the other. 

why don't we use this opportunity to split the appointment in the DTO?

Reply:
i.e. 

{
  id: ...
  appointment: {
    durationInMinutes: ...,
    appointmentTime: ...,
    createdBy: ...,
    deliusAppointmentId: ...,
    ...
    }
 }